src/utils/presence.js

```diff
@@ -0,0 +1,40 @@
+/* @flow */
+
+const OFFLINE_THRESHOLD_SECS = 140;
+
+export const getAggregatedPresence = (presence: Object) => {
+  let aggregatedStatus = 'offline';
+  let aggregatedTimestamp = 0;
+  let aggregatedClient = '';
+  Object.keys(presence).forEach(client => {
```

Comment:
Use `filter` to remove `aggregated`, then use `map` or `reduce` to create and return a new object.

Do not forget to add the proper Flow types!